docs_src/design/adr/system-management/001-docker-image-guidelines.md

```diff
@@ -0,0 +1,34 @@
+# Docker image guidelines
+
+## Status
+
+Pending
+
+## Context
+
+When deploying the EdgeX Docker containers some security measures are recommended to ensure the integrity of the software stack.
+
+## Decision
+
+When deploying Docker images, the following flags should be set for heightened security.
+
+- To avoid escalation of privileges each docker container should use the `--security-opt=no-new-privileges` flag. More details about this flag can be found [here](https://docs.docker.com/engine/reference/run/#security-configuration)
```

Comment:
EdgeX has no control over the user's docker daemon.  We are only allowed to exercise control via Dockerfile or docker-compose.  In this case, we want

```
    security_opt:
      - "no-new-privileges:true"
```